### ISSUE TITLE
"shares"  empty from friendica in hubzilla

### ISSUE THREAD

**Issue by elvecio** [friendica/friendica#4984]
in truth I did not know whether to post here or in hubzilla, but as I have noticed that the problem comes exactly from friendica, it is probably here that I have to write.
I am in hubzilla, and the normal posts that come from friendica do not have problems, while all posts that come from friendica but that are actually "shares" of other posts, arrive **empty**, just with this writing:

_Mister x "wrote the following post about x hours ago"_

just this.
Other "shares" (from diaspora or socialhome  for ex.) arrive without problems, is just friendica the problem.

**Comment by MrPetovan** [comment]
Can you please add a public URL of an empty Friendica share?

**Comment by elvecio** [comment]
for ex. _reshares_ posts from this person https://squeet.me/profile/perig in the last week

**Comment by MrPetovan** [comment]
Thank you, can you add a Hubzilla link to one of his reshares? I don't have a Hubzilla account to test myself.

**Comment by elvecio** [comment]
no, hubzilla has original source links, so the same you can find in https://squeet.me/profile/perig

**Comment by annando** [comment]
This should be fixed by now.
